tests/unit/linalg/test_hypre_vector.cpp

```diff
@@ -17,18 +17,12 @@ namespace mfem
 
 #ifdef MFEM_USE_MPI
 
-TEST_CASE("HypreParVector I/O", "[Parallel], [HypreParVector]")
+namespace detail
```

Comment:
We use `internal` to hide some implementation details in Doxygen, see https://github.com/mfem/mfem/blob/167ff72edccc4e5b0a642ae3392ccaf50f455b9e/doc/CodeDocumentation.conf.in#L857
You can also add `mfem::detail` there if you want to keep the `detail` namespace here and exclude it from Doxygen.